lib/presenter/printable_list.rb

```diff
@@ -0,0 +1,21 @@
+module Presenter
+  class PrintableList
+    def initialize(title:)
+      @title = title
+    end
```

Comment:
We can easily choose between `X visits`, `Y unique views` or whatever else in our output.